### PR TITLE
[all-permissions] Support upcoming web-wide clipboard copy API.

### PR DESCRIPTION
**Pull request by lgarron**
... since it's already landed in Chrome Canary.

(I found myself a little bit lost talking about the API on Tuesday without actually trying it out.)

### CODE REVIEW

File: demos/all-permissions.html [hunk 1]
```diff
@@ -136,6 +141,24 @@
       displayOutcome("midi", "error")
     );
   },
+  "copy": (function() {
+    var interceptCopy = false;
+
+    document.addEventListener("copy", function(e){
+      if (interceptCopy) {
+        // From http://www.w3.org/TR/clipboard-apis/#h4_the-copy-action
+        e.clipboardData.setData("text/plain", "This text was copied from the all-permissions demo.");
```
Comment (adrifelt) [inline demos/all-permissions.html]: friends don't let friends skip line wrap day

Comment (lgarron) [inline demos/all-permissions.html]: Alright. What's your favorite multiline JS comment mechanism?

Comment (lgarron) [inline demos/all-permissions.html]: PR updated to use string concatenation with `+`.

Comment (adrifelt) [inline demos/all-permissions.html]: danke schoen

**Comment by adrifelt** [comment]
lgtm

**Pull request merged**